CMakeLists.txt

```diff
@@ -123,6 +126,9 @@ endif()
 if(Cabana_ENABLE_HDF5 AND NOT Cabana_ENABLE_MPI)
   message(FATAL_ERROR "Cabana HDF5 support requires MPI.")
 endif()
+if(NOT HDF5_IS_PARALLEL)
+  message(FATAL_ERROR "Cabana HDF5 support requires parallel HDF5.")
+endif()
 
```

Comment:
Changed check for HDF5_IS_PARALLEL to: 

  include(CheckSymbolExists)
  check_symbol_exists(H5_HAVE_PARALLEL "H5pubconf.h" HDF5_IS_PARALLEL)

cc: @junghans

Reply:
I'm not seeing any output from the find HDF5 package.

Reply:
You're right - I updated the new check for only the cases where HDF5 is enabled